docs/Running-paie.rst

```diff
@@ -175,3 +173,24 @@ NOTES:
 
 After completion of the installation of the PowerAI Enterprise software, you must configure Spectrum Conductor Deep Learning Impact and apply any outstanding fixes.
 Go to https://www.ibm.com/support/knowledgecenter/SSFHA8, choose your version of PowerAI Enterprise and then use the search bar to search for ‘Configure IBM Spectrum Conductor Deep Learning Impact’.
+
+Additional Notes
+~~~~~~~~~~~~~~~~
+
+You can browse the content of the POWER-Up software server by pointing a web browser
+at the POWER-Up installer node. Individual files can be copied to client nodes using wget or
+curl if desired.
+
+**Dependent software packages**
+The PowerAI Enterprise software is dependent on additional open source software that is not shipped with PowerAI Enterprise.
+Some of these dependent packages are downloaded to the POWER-Up software server from enabled yum repositories during the preparation phase and are subsequently available to the client nodes during the install phase.
+Additional software packages can be installed in the 'dependencies' repo on the POWER-Up software server by listing them in the power-up/software/dependent-packages.list file.
+Entries in this file can be delimited by spaces or commas and can appear on multiple lines.
+Note that packages listed in the dependent-packages.list file are not automatically installed on client nodes unless needed by the PowerAI software.
+They can be installed on a client node explicitly using yum on the client node (ie yum install pkg-name). Alternatively, they can be installed on all client nodes at once using Ansible (run from within the power-up/software/ directory)::
+
+    $ ansible all -i software_hosts -m yum -a "name=pkg-name"
+
```

Comment:
If running from 'power-up/software/' they will need '-i ../playbooks/software_hosts'. This should probably be moved into the software directory though.